tests/test_entry.py

```diff
@@ -32,7 +61,14 @@ def test_modify_entry_with_put_successfully(self):
         entry_data = Entry.json(self.entry)
         self.entry.entryId = 1
         put_url = self.client.put('api/v1/entries/{}'.format(self.entry.entryId),data=entry_data,content_type='application/json')
-        self.assertEqual(put_url.status_code, 201)
+        self.assertEqual(put_url.status_code, 200)
+
+    def test_delete_entry_with_delete_successfully(self):
+        entry_data = Entry.json(self.entry)
+        self.entry.entryId = 1
+        delete_url = self.client.delete('api/v1/entries/{}'.format(self.entry.entryId),data=entry_data,content_type='application/json')
```

Comment:
This line is too long.
Its better to have a maximum line width of 100 characters to improve readability